sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/types/Resolver.kt

```diff
@@ -139,7 +203,8 @@ class Resolver(private val symbolTable: SymbolTable) {
           "Cannot find table or view ${tableName.text}")
       if (!currentlyResolvingViews.add(view.view_name().text)) {
         throw SqlitePluginException(view.view_name(),
-            "Recursive subquery found: ${currentlyResolvingViews.joinToString(" -> ")} -> ${view.view_name().text}")
+            "Recursive subquery found: ${currentlyResolvingViews.joinToString(
+                " -> ")} -> ${view.view_name().text}")
```

Comment:
It's pretty insane that this is valid syntax in Kotlin. You've put a line break inside a string but in a place that doesn't actually affect the string itself. I think I'd prefer the complex expression be done in a local var and then the string just contain the `$localVar` where it needs to be.